regen_go_proto_canonical_protos.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+bazelisk build //proto/cel/expr:all
+files=($(bazelisk aquery 'kind(proto, //proto/cel/expr:all)' | grep Outputs | grep "[.]pb[.]go" | sed 's/Outputs: \[//' | sed 's/\]//' | tr "," "\n"))
+for src in ${files[@]};
+do
+  dst=$(echo $src | sed 's/\(.*\%\/github.com\/google\/cel-spec\/\(.*\)\)/\2/')
+  $(cp $src ./go)
```

Comment:
oh - this needs to be fixed, since there's no go directory anymore.

Reply:
I think you might've been looking at this before I pushed the latest commit :) (same with below)